Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -278,6 +278,7 @@ extension SPAppDelegate: SimperiumDelegate {
 
     public func simperium(_ simperium: Simperium, didFailWithError error: Error) {
         SPTracker.refreshMetadataForAnonymousUser()
+        logOutIfAccountDeletionRequested()
```

Comment:
We should only handle `SPSimperiumErrorsInvalidToken` error here as that's the only case we're interested in

Reply:
+1 to this one 😄